src/main/java/com/alkemy/ong/exception/GlobalControllerExceptionHandler.java

```diff
@@ -0,0 +1,28 @@
+package com.alkemy.ong.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import java.time.ZonedDateTime;
+
+@ControllerAdvice
+public class GlobalControllerExceptionHandler {
+
+
+    @ExceptionHandler
```

Comment:
Could you please add the exception handler only for this particular exception MethodArgumentNotValidException.class. Like:

```java
@ExceptionHandler(MethodArgumentNotValidException.class)
```